src/resources/PowerShellWorkerStrings.resx

```diff
@@ -229,7 +229,7 @@
   <data name="FailToCreateFunctionAppDependenciesDestinationPath" xml:space="preserve">
     <value>Fail to create FunctionApp dependencies destination path '{0}'. Please make sure you have write access to this location. Error '{1}''. </value>
   </data>
-    <data name="FailToInstallFuncAppDependency" xml:space="preserve">
-    <value>Fail to install FunctionApp dependent module '{0}' version '{1}'. Error: '{2}''. </value>
+    <data name="FailToInstallModule" xml:space="preserve">
+    <value>Fail to install module '{0}' version '{1}'. Error: '{2}'</value>
```

Comment:
Better add the info about the number of attempts, like `Fail to install module 'Az' version '2.0'. 1st attempt. Error: '{3}'`

Reply:
Thanks @daxian-dbw for the suggestion--I have added the current attempt to the error message. Could you please take another look? Thanks!